v2/pkg/release/collector_test.go

```diff
@@ -14,6 +14,27 @@ import (
 	"github.com/openshift/oc-mirror/v2/pkg/mirror"
 )
 
+// setup mocks
+// we need to mock Manifest, Mirror, Cincinnati
+
+type Mirror struct {
```

Comment:
Nit: 
Suggestion to rename these as MockMirror, MockManifest & MockCincinnati?
Since they are now at the top, a bit away from the interface method implementations, it will be easier to understand?

Reply:
Done, thanks @sherine-k